docs/operators/fixedWidthParserOperator.md

```diff
@@ -0,0 +1,170 @@
+Fixed Width Parser Operator
+=============
+
+## Operator Objective
+This operator is designed to parse fixed width records and construct a map or concrete java class also known as ["POJO"](https://en.wikipedia.org/wiki/Plain_Old_Java_Object) out of it. User need to provide the schema to describe the fixed width data. Based on schema definition the operator will parse the incoming record to object map and POJO.
+Valid records will be emitted as POJO / map while invalid ones are emitted on error port with error message.
```

Comment:
Revise: Invalid records will be emitted on the error port along with an error message.